services/apps/src/apps/routes/domain/app/category.clj

```diff
@@ -83,3 +85,25 @@
   (-> AppCategoryRequest
       (->optional-param :name)
       (->optional-param :parent_id)))
+
+(defschema AppCategoryOntologyVersionDetails
+  {:version    (describe String "The unique version of the Ontology")
+   :applied_by (describe NonBlankString "The user that set this version as active")
+   :applied    (describe Date "The date this version was set as active")})
```

Comment:
Something to check -- many of our things return dates as integers (unix timestamps, I believe). I don't know if it'd potentially be worth maintaining that compatibility, or if I'm misremembering, or if we may as well just start doing this like this instead.